Formula/redis.rb

```diff
@@ -20,7 +20,7 @@ class Redis < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "9a60623e87e59fa76b87a32f036430e1379a5c19fe4c48151b42a2d5aa3c0032"
   end
 
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
```

Comment:
```suggestion
  depends_on "openssl@1.1"
```
We're stuck on this for now.

Reply:
Changed, yes still stuck due to other dependencies, despite Redis being compatible with OpenSSL 3.0.